depmapomics/expressions.py

```diff
@@ -557,10 +567,14 @@ async def postProcess(
 
     if compute_enrichment:
         print("doing ssGSEA")
-        enrichments = await ssGSEA(files[ssGSEAcol], recompute=recompute_ssgsea)
+        enrichments = await _ssGSEA(
+            files[ssGSEAcol], ssgseafilepath, recompute=recompute_ssgsea
+        )
         print("saving files")
         enrichments.to_csv(save_output + "gene_sets_all.csv")
-    saveFiles(files, save_output)
```

Comment:
what do you think of keeping some functions as non private (the ssGSEA one for example and subset_genes and prot_cod genes, I would run outside of releases).

Reply:
Yes I think it's reasonable to keep them public. I don't have a good sense of what other functions should be kept public though. Anything other than these 3?